libsolutil/Whiskers.cpp

```diff
@@ -36,10 +36,16 @@ Whiskers::Whiskers(string _template):
 {
 }
 
+void Whiskers::setTemplate(string _template)
+{
+	m_template = move(_template);
+}
+
 Whiskers& Whiskers::operator()(string _parameter, string _value)
 {
 	checkParameterValid(_parameter);
 	checkParameterUnknown(_parameter);
+	checkTemplateContainsTags(_parameter, {""});
```

Comment:
It can also just contain `+paramName`.

Reply:
Sure, but the errors are only generated for tags that must be present in the template.